pytest_html/plugin.py

```diff
@@ -112,8 +113,10 @@ def __init__(self, outcome, report, contained_html, logfile):
             self.contained_html = contained_html
             self.logfile = logfile
 
+            extra_index = 1
             for extra in getattr(report, 'extra', []):
```

Comment:
If you use [`enumerate`](https://docs.python.org/2.7/library/functions.html#enumerate) then you will be able to access the index.